tools/UpdatePaths/Scripts/72024_hos_beret.txt

```diff
@@ -0,0 +1 @@
+/obj/item/clothing/head/hos/beret/ : /obj/item/clothing/head/hats/hos/beret{@OLD}
```

Comment:
```suggestion
/obj/item/clothing/head/hos/beret : /obj/item/clothing/head/hats/hos/beret{@OLD}
```

Whoops, that didn't apply on the commit I suggested. I think it's because I edited the suggestion, so it didn't apply when you hit the button (within a minute of me editing it)? I'm just going to pass it in now because it won't work like this.